app/views/browse/deployment-config.html

```diff
@@ -47,12 +47,12 @@
                       <a ng-href="project/{{projectName}}/set-limits?dcName={{deploymentConfig.metadata.name}}"
                         role="button">Set Resource Limits</a>
                     </li>
-                    <li ng-if="!autoscalers.length && ('extensions/horizontalpodautoscalers' | canI : 'create')">
+                    <li ng-if="!autoscalers.length && ({resource: 'horizontalpodautoscalers', group: 'autoscaling'} | canI : 'create')">
```

Comment:
Generally, as a rule of thumb, we should use the appropriate and stable groups for resources instead of the `extensions` one. The resources in the latter are meant for deprecation sooner or later, whereas the `autoscaling` in case of HPA will keep working. The same applies to other group/resource pairs like `batch/Job`, etc.